ui/.eslintrc.js

```diff
@@ -11,10 +11,11 @@ const dependencies = getDependencies();
 
 module.exports = createConfig({
   cwd: __dirname,
-  incrementalAdoption: true, // turn everything into a warning
   rules: {
-    'unicorn/consistent-destructuring': 'off', // too many failures in React class components
-    'promise/catch-or-return': 'off', // requires deeper refactoring of fetching layer
+    'import/no-default-export': 'off', // too opinionated
+    'import/no-unresolved': 'off', // lots of false positives (perhaps misconfigured?)
+    'simple-import-sort/imports': 'off', // changes global CSS order; must migrate to CSS modules first
```

Comment:
Another reason why global CSS is bad (especially when imported from each component file): it is hugely dependent on the order in which components are used/imported in the application.

Reply:
:)